app/src/app/close.js

```diff
@@ -49,8 +52,16 @@ export default function(e) {
     }
     if (isOSX()) app.hide()
   }
+  const lastUrl = state.mainWindow.webContents.getURL()
+  const isLastUrlChat = (
```

Comment:
I talked with @sk7 and probably in a follow up this check should be about checking if there is an active section for the user. Otherwise it doesn't make much sense to save secondary pages like log out